### DISCUSSION TITLE
Validation?

### DISCUSSION THREAD

**Discussion by jonas2711** [xdan/jodit#891] [Q&A] [answered]
Is it possible to add default HTML validation (required), and how?

**Comment by ketna11** [comment]
If you use Bootstrap, you need to change textareas from display: none; to opacity: 0; Validation will start working.

**Comment by xdan** [comment, marked as answer]
Native `required` validation doesn't fire because Jodit hides the original textarea with `display:none`, and browsers skip validation on display:none fields. The tip from @ketna11 above is the fix — make the textarea `opacity: 0` (zero-size/positioned absolutely) instead of `display:none`, so the browser still validates it. Otherwise just validate `editor.value` yourself on submit.
